components/InfoPane/InfoPane.tsx

```diff
@@ -37,6 +37,31 @@ export default function InfoPane(props: HTMLProps<HTMLDivElement>) {
           <FilePicker label="Choose file" /> from your computer
         </li>
       </ul>
+
+      <hr />
+      <footer>
+        <p>
+          npmgraph
+          {' | '}
+          <a
+            href="https://github.com/npmgraph/npmgraph"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="bright-hover external-link"
+          >
+            GitHub repo
+          </a>
+          {' | '}
+          <a
+            href="https://github.com/sponsors/broofa"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="bright-hover external-link"
+          >
+            Sponsor
+          </a>
```

Comment:
Selfishly, I'd prefer to keep this as the `GithubSponsorButton`.  It's fine for now, but don't be surprised if I revert back to that at some point, just FYI. :-)

Reply:
Agreed that having a link and button side-by-side isn't a great look.   'Could probably style them both as buttons that are slightly smaller in size as a happy compromise.

Problem for a later date.